main.go

```diff
@@ -39,13 +39,14 @@ import (
 
 type contextKey string
 
-const contextKeyError = contextKey("error")
-
-const cloudPlatformScope = "https://www.googleapis.com/auth/cloud-platform"
-
-const Version = "0.1.0"
-const OSArch = runtime.GOOS + "/" + runtime.GOARCH
-const UserAgent = "cloud-run-proxy/" + Version + " (" + OSArch + ")"
+const (
```

Comment:
I actually had these grouped separately on purpose. Can you revert?

Reply:
done